src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -491,6 +491,7 @@ internal enum WellKnownMember
         System_Span_T__get_Item,
         System_Span_T__get_Length,
         System_Span_T__Slice_Int_Int,
+        System_Span_T__op_Implicit_Span,
```

Comment:
I hate this name. I know it follows our standard convention, but I was confused reading the lowering code because it took me a minute to remember that this returns a `ReadOnlySpan`.

Reply:
We could include the target type in the name, perhaps `System_Span_T__op_Implicit_ReadOnlySpan_Span`.